src/Lykke.MatchingEngine.Connector/Services/TcpOrderSocketService.cs

```diff
@@ -11,40 +11,61 @@ public class TcpOrderSocketService : ITcpClientService, ISocketNotifyer
         private readonly TasksManager<long, TheResponseModel> _tasksManager;
         private readonly TasksManager<string, TheNewResponseModel> _newTasksManager;
         private readonly TasksManager<string, MarketOrderResponseModel> _marketOrdersTasksManager;
+        private readonly ISocketLog _logger;
+        private readonly bool _ignoreErrors;
 
         public TcpOrderSocketService(TasksManager<long, TheResponseModel> tasksManager,
             TasksManager<string, TheNewResponseModel> newTasksManager,
-            TasksManager<string, MarketOrderResponseModel> marketOrdersTasksManager)
+            TasksManager<string, MarketOrderResponseModel> marketOrdersTasksManager,
+            ISocketLog logger = null,
+            bool ignoreErrors = false)
         {
             _tasksManager = tasksManager;
             _newTasksManager = newTasksManager;
             _marketOrdersTasksManager = marketOrdersTasksManager;
+            _logger = logger;
+            _ignoreErrors = ignoreErrors;
         }
 
         public async Task HandleDataFromSocket(object data)
         {
             await Task.Run(() =>
             {
-                var theResponse = data as TheResponseModel;
-                if (theResponse != null)
+                try
                 {
-                    _tasksManager.Compliete(theResponse.ProcessId, theResponse);
-                    Console.WriteLine($"Response ProcessId: {theResponse.ProcessId}");
-                    return;
-                }
+                    var theResponse = data as TheResponseModel;
+                    if (theResponse != null)
+                    {
+                        _logger?.Add($"Response ProcessId: {theResponse.ProcessId}");
+                        _tasksManager.Compliete(theResponse.ProcessId, theResponse);
+                        return;
+                    }
 
-                var theNewResponse = data as TheNewResponseModel;
-                if (theNewResponse != null)
-                {
-                    _newTasksManager.Compliete(theNewResponse.Id, theNewResponse);
-                    Console.WriteLine($"Response Id: {theNewResponse.Id}");
-                }
+                    var theNewResponse = data as TheNewResponseModel;
+                    if (theNewResponse != null)
+                    {
+                        _logger?.Add($"Response Id: {theNewResponse.Id}");
+                        _newTasksManager.Compliete(theNewResponse.Id, theNewResponse);
```

Comment:
Compliete or Complete?

Reply:
It is a typo in a Lykke.Common library. Would definitely fix it when I got to that code :)